base/initdefs.jl

```diff
@@ -19,19 +19,22 @@ An array of the command line arguments passed to Julia, as strings.
 const ARGS = String[]
 
 """
-    exit([code])
+    exit(code=0)
+
+Quit the program with an exit code. The default exit code is zero, indicating that the
+program completed successfully (see also [`quit`](@ref)). In an interactive session,
+`exit()` can be called by typing `^D`.
```

Comment:
Perhaps remove `typing` since you are not really typing anything? :) Same for `quit`